package.json

```diff
@@ -135,6 +134,7 @@
     "lodash.shuffle": "^4.2.0",
     "observable-hooks": "^4.2.1",
     "qrcode": "^1.5.3",
+    "rc-table": "^7.48.0",
```

Comment:
Where is this used, what tables?

Reply:
`import { GetRowKey } from 'rc-table/lib/interface'`